Backend/server.js

```diff
@@ -35,11 +35,12 @@ connection.once('open', () => {
 //first route
 const showRouter = require('./routes/shows');
 const platformRouter = require('./routes/platform');
+const userRouter = require('./routes/user');
 const watchlistRouter = require('./routes/watchlist');
```

Comment:
we might have issues with this import since the file watchlist doesn't exist. if it runs then it's okay but if it doesn't then you can just make a blank watchlist.js file

Reply:
That's strange the watchlist line isn't in my code were there any other pull requests merged today?